packages/core/utils/package.json

```diff
@@ -40,6 +40,7 @@
     "dotenv": "^16.4.5",
     "jsonwebtoken": "^9.0.2",
     "knex": "2.4.2",
+    "pluralize": "^8.0.0",
```

Comment:
Q: what is the impact with our current pluralize type compared to real output?

Reply:
Worst case scenario I will have to fix tests/Interfaces/Modules where we have wrong plurals.
What we might have to revisit is the TypeScript types that won't be correct

Reply:
Yes thats definitely my point, the type need to match the function rules, we can check that separately